javascript/code-challenges/tree/tree.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
```

Comment:
You are off to such a great start Nathan, I know that this was a struggle given the holiday weekend. When you have some time to go back to this, I will be here to regrade and leave comments as much as needed.